cirq-aqt/README.rst

```diff
@@ -0,0 +1,27 @@
+.. image:: https://quantumai.google/site-assets/images/marketing/icons/ic-qcs.png
+  :target: https://github.com/quantumlib/cirq/
+  :alt: cirq-aqt
+  :width: 500px
+
+`Cirq <https://quantumai.google/cirq>`__ is a Python library for writing, manipulating, and optimizing quantum
+circuits and running them against quantum computers and simulators.
+
+This module is **cirq-aqt**, which provides everything you'll need to run Cirq quantum algorithms on with Alpine Quantum quantum computers.
```

Comment:
nit: Alpine Quantum quantum sounds silly.  Maybe change to "Alpine Quantum devices"?

Reply:
Yeah, or AQT